File: replace_problematic_files.py
#!/usr/bin/env python3
import os
import re
import glob

def create_simple_page(file_path):
    """Create a simple, working page for problematic files"""
    try:
        # Extract the service name from the path
        path_parts = file_path.split('/')
        service_name = path_parts[-2] if path_parts[-1] == 'page.tsx' else path_parts[-1].replace('.tsx', '')
        
        # Convert service name to proper format
        display_name = service_name.replace('-', ' ').replace('_', ' ').title()
        function_name = service_name.replace('-', '').replace('_', '').replace(' ', '')
        
        # Create a simple working page
        content = f'''import React from 'react';
import {{ Helmet }} from 'react-helmet-async';
import {{ Link }} from 'react-router-dom';
import {{ ArrowRight }} from 'lucide-react';

export default function {function_name}Page() {{
  return (
    <div className="min-h-screen bg-gradient-to-br from-slate-900 via-purple-900 to-slate-900 pt-20">
      <Helmet>
        <title>{display_name} - Zion Tech Group</title>
        <meta name="description" content="Professional {display_name} services by Zion Tech Group. Transform your business with our expert solutions." />
      </Helmet>
      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8 py-16 text-center">
        <h1 className="text-4xl font-bold text-white mb-6">{display_name}</h1>
        <p className="text-lg text-gray-300 mb-8">Professional {display_name} services coming soon.</p>
        <Link
          to="/contact"
          className="bg-gradient-to-r from-cyan-500 to-purple-600 text-white px-8 py-4 rounded-lg font-semibold hover:from-cyan-600 hover:to-purple-700 transition-all duration-300 flex items-center justify-center mx-auto w-fit"
        >
          Contact Us
          <ArrowRight className="w-5 h-5 ml-2" />
        </Link>
      </div>
    </div>
  );
}}'''
        
        with open(file_path, 'w', encoding='utf-8') as f:
            f.write(content)
        
        print(f"Replaced {file_path} with simple page")
        return True
        
    except Exception as e:
        print(f"Error replacing {file_path}: {e}")
        return False

def main():
    # List of problematic files to replace
    problematic_files = [
        'app/5g-edge-computing/page.tsx',
        'app/5g-implementation/page.tsx',
        'app/5g-iot-solutions/page.tsx',
        'app/5g-mobile-applications/page.tsx',
        'app/5g-network-infrastructure/page.tsx',
        'app/5g-private-networks/page.tsx',
        'app/5g-smart-city-solutions/page.tsx',
        'app/5g-solutions/page.tsx',
        'app/about/page.tsx',
        'app/accessibility-page/page.tsx',
        'app/accessibility/page.tsx',
        'app/advanced-security-suite/page.tsx',
        'app/ai-3d-generation/page.tsx',
        'app/ai-accounting-assistant/page.tsx',
        'app/ai-agricultural-intelligence-pro/page.tsx',
        'app/ai-analytics-dashboard-pro/page.tsx',
        'app/ai-analytics-dashboard/page.tsx',
        'app/ai-analytics/page.tsx'
    ]
    
    replaced_count = 0
    for file_path in problematic_files:
        if os.path.exists(file_path):
            if create_simple_page(file_path):
                replaced_count += 1
    
    print(f"Replaced {replaced_count} problematic files with simple working versions")

if __name__ == "__main__":
    main()

File: replace-problematic-files.js
const fs = require('fs');
const path = require('path');

// Read the template
const template = fs.readFileSync('./template-page.tsx', 'utf8');

// List of problematic files to replace
const problematicFiles = [
  'app/accessibility-page/page.tsx',
  'app/accessibility/page.tsx',
  'app/ad-management/page.tsx',
  'app/advanced-security-suite/page.tsx',
  'app/ai-accounting-assistant/page.tsx'
];

function replaceFile(filePath) {
  try {
    // Get the directory name to create a proper title
    const dirName = path.dirname(filePath).split('/').pop();
    const title = dirName.replace(/-/g, ' ').replace(/\b\w/g, l => l.toUpperCase());
    
    // Replace the title in the template
    const content = template.replace('Page - Zion Tech Group', `${title} - Zion Tech Group`);
    
    // Write the file
    fs.writeFileSync(filePath, content, 'utf8');
    console.log(`Replaced: ${filePath}`);
    return true;
  } catch (error) {
    console.error(`Error replacing ${filePath}:`, error.message);
    return false;
  }
}

console.log('Replacing problematic files...');
let replacedCount = 0;

problematicFiles.forEach(file => {
  if (replaceFile(file)) {
    replacedCount++;
  }
});

console.log(`Replaced ${replacedCount} files`);